src/System.Net.Sockets/tests/FunctionalTests/DualModeSocketTest.cs

```diff
@@ -1056,6 +1056,7 @@ private void DualModeConnect_AcceptAsync_Helper(IPAddress listenOn, IPAddress co
         }
     }
 
+    [OuterLoop] // https://github.com/dotnet/corefx/issues/17681
```

Comment:
Nit: OuterLoop has a `string reason` ctor... that could be used instead of the comment.

Reply:
Thanks! Didn't know that.  I'll use that next time.